yarn-project/simulator/src/public/avm/serialization/bytecode_serialization.ts

```diff
@@ -50,13 +50,10 @@ import {
 } from '../opcodes/index.js';
 import { BufferCursor } from './buffer_cursor.js';
 import { MAX_OPCODE_VALUE, Opcode } from './instruction_serialization.js';
+import type { Serializable } from './serializable.js';
 
 export type InstructionDeserializer = (buf: BufferCursor | Buffer) => Instruction;
 
-export interface Serializable {
-  serialize(): Buffer;
-}
-
 export interface Deserializable {
```

Comment:
may aswell move Deserializable into the same file and it just becomes the interface.ts file for serialization

Reply:
bruh.

Reply:
in that case serialisable could do with a DONT TOUCH THIS comment. I imagine the another gardener will have the same idea and not think of it